Extensions/AzureAdAuthenticationBuilderExtensions.cs

```diff
@@ -1,13 +1,15 @@
-﻿using System;
-using Microsoft.AspNetCore.Authentication.OpenIdConnect;
-using Microsoft.Extensions.Configuration;
+﻿using Microsoft.AspNetCore.Authentication.OpenIdConnect;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Options;
+using Microsoft.Identity.Client;
+using System;
+using System.Threading.Tasks;
+using TodoListService.Extensions;
```

Comment:
Be consistent on ordering of 'usings'
usually 'System' is first

Reply:
Made sure the 'using System...' statements were first in all the files in the sample.